File: 217-contains-duplicate/217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
#       create hash
        hashset = set()
        for n in nums:
            if n in hashset:
                return True
#               if it doesn't contains duplicate we add value
            hashset.add(n)
        return False
                
                
                
# 3 different ways to solve
# brute force , sorting and using hashset

File: 217-contains-duplicate/217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
// create hashset
//    first sort the numbers and loop through it 
    nums.sort((a,b)=> {return b-a});
    
    for (let i = 0 ; i < nums.length ; i++){
        if(i > 0 && nums[i-1] === nums[i]){
            return true
        }
      
        
    }
      return false
};